### ISSUE TITLE
Parameters with 'content' are not supported

### ISSUE THREAD

**Issue by npdev453** [horiuchi/dtsgenerator#472]
There is an endpoint with parameter: `lastName` that's described using `content`

Looks like `parameters/*/content/[media-type]` are not supported [SupportedFeatures](https://github.com/horiuchi/dtsgenerator/blob/master/SupportedFeatures.md#openapi).

After running `dtsgen ./input.yaml` command does not return `Parameters` at all.

> input.yaml
```
openapi: "3.0.0"
info:
  title: ParamRefExample
  version: 1.0.0
paths:
  "/endpoint/two/":
    get:
      operationId: operationOne
      parameters:
        - name: lastName
          in: query
          content: 
            application/json: 
              schema: 
                type: string
      responses:
        200:
          description: Success
```
> stdout
```
declare namespace Paths {
    namespace OperationOne {
        namespace Responses {
            export interface $200 {
            }
        }
    }
}
```

Based on: https://github.com/anttiviljami/openapi-client-axios/issues/40

**Comment by horiuchi** [comment]
Released by v3.10.0

**Comment by sebmasc** [comment]
@npdev453 can You check if its working ? 
In my case, i have still empty parameters.
